com.wso2telco.dep.hub.creditapi/.project

```diff
@@ -5,8 +5,14 @@
 	<projects>
 	</projects>
 	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
 	</buildSpec>
 	<natures>
+		<nature>org.eclipse.m2e.core.maven2Nature</nature>
 		<nature>org.wso2.developerstudio.eclipse.mavenmultimodule.project.nature</nature>
 	</natures>
```

Comment:
These change are for credit API. please check what are the file you want to add for a commit. 
additionally you don't need these changes to add.